src/app/clients/ClientsPage.tsx

```diff
@@ -166,6 +168,7 @@ const ClientsPage: React.FC<Props> = (props) => {
                     columnStyleOptions={clientTableColumnStyleOptions}
                     onRowClick={onClientTableRowClick}
                     checkboxes={true}
+                    onRowSelection={setSelected}
```

Comment:
how does this work with pagination? can you select rows 1 and 20 if only 10 are shown at once?

Reply:
With the current implementation of pagination it works fine (it uses rowIdx which is always unique even between pages) but this might break once @paopaosoftwire changes how pagination works